nxrm-ha/Chart.yaml

```diff
@@ -15,7 +15,7 @@ type: application
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 83.0.0
+version: 83.0.1
```

Comment:
Don't change this file - it will automatically get written when we publish a release. We can include it in 3.83.1, but revert this change first.

Reply:
Sure thanks Nick.